app/src/main/res/layout/player.xml

```diff
@@ -216,7 +216,7 @@
                         android:paddingEnd="6dp"
                         android:paddingBottom="3dp"
                         android:scaleType="fitCenter"
-                        android:src="@drawable/ic_format_list_numbered"
```

Comment:
format_list_numbered doesn't seem to be used anymore now. Please remove it

Reply:
It has been removed now